cmd/salad-http-job-queue-worker/main.go

```diff
@@ -18,7 +19,19 @@ import (
 	"github.com/saladtechnologies/salad-cloud-job-queue-worker/pkg/workers"
 )
 
+var VerFlag bool
+
+func init() {
+	flag.BoolVar(&VerFlag, "version", false, "Show version info")
+}
+
 func main() {
+	flag.Parse()
+	if VerFlag {
+		fmt.Printf("Version: %s\n", workers.Version)
```

Comment:
This only prints the SemVer or `v0.0.0` if the build is from a non-tagged commit. This is not the same value that we log when running the worker which includes the short Git commit hash. Should this be updated to match what we print when running the worker?

Reply:
I believe so.  To things that are not the backend, v0.0.0 is not really useful in determining the pedigree of the binary.